README.md

```diff
@@ -40,25 +40,25 @@ For more information see "Ignore-Registry" and "Ignore-Image-Name" tokens in the
 4. Aggregation policy: You can aggregate multiple scan results in a single ticket/message. This is useful if you would like to get a digest on daily/weekly basis.
 For more information see "Aggregate-Issues-Number" and "Aggregate-Issues-Timeout" tokens in the cfg.yaml file.
 
-## Quick Start 
+## Quick Start
 Follow these steps to set up JIRA integration:
 
-1. Clone this project: 
+1. Clone this project:
 ```bash
 git clone git@github.com:aquasecurity/postee.git
 ```
 
-2. Build the postee Docker image: 
+2. Build the postee Docker image:
 ```bash
 docker build -t postee:latest .
```

Comment:
Can you change the image name to aqua-postee? I see that in the run command you aree using aqua-postee.